static/app/views/issueList/issueListTable.tsx

```diff
@@ -62,9 +66,21 @@ function IssueListTable({
   paginationAnalyticsEvent,
   personalSavedSearches,
   organizationSavedSearches,
+  issuesSuccessfullyLoaded,
 }: IssueListTableProps) {
+  const location = useLocation();
   const {newViewActive} = useContext(NewTabContext);
 
+  const isNewViewEmptyStateActive =
+    isNewViewPage(location.pathname) &&
```

Comment:
I wish I didn't have to check the pathname here. A better solution would be to break up overview.tsx so that I can render things differently on the new view route, but that refactor would have been huge.